src/audio-worklet/free-queue/src/interface/README.md

```diff
@@ -1,8 +1,9 @@
 # FreeQueue C Interface
 
 A single header C API implementation to access FreeQueue functionality from C/C++ programs.
-
-## Using
+This uses atomics from `stdatomic.h` and emscripten to implement atomic read and writes.
```

Comment:
Please wrap the line at column 80. Make it consistent with other README.md files.

Reply:
Okay!!